File: Vincent OS/Login2bureau2.vb
Public Class Login2bureau2

    Private Sub Login2bureau2_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Timer1.Stop()
        ProgressBar1.Value = 0
        Timer1.Start()
    End Sub

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        ProgressBar1.Increment(3)
        If ProgressBar1.Value = 100 Then
            Timer1.Stop()
            ProgressBar1.Value = 0
            Bureau2.Show()
            Me.Close()
        End If
    End Sub
End Class

File: src/Vincent OS/Login2bureau2.cs
using System;

namespace Vincent_OS
{
    public partial class Login2bureau2
    {
        public Login2bureau2()
        {
            InitializeComponent();
        }

        private void Login2bureau2_Load(object sender, EventArgs e)
        {
            Timer1.Stop();
            ProgressBar1.Value = 0;
            Timer1.Start();
            this.WindowState = System.Windows.Forms.FormWindowState.Maximized;
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            ProgressBar1.Increment(3);
            if (ProgressBar1.Value == 100)
            {
                Timer1.Stop();
                ProgressBar1.Value = 0;
                My.MyProject.Forms.Bureau2.Show();
                Close();
                GC.Collect();
                GC.WaitForPendingFinalizers();
            }
        }
    }
}